.gitignore

```diff
@@ -1,3 +1,5 @@
 _bin/*
 .vs/*
-*.user
\ No newline at end of file
+*.user
+bin
```

Comment:
Is it change required? All build files should be produced to _bin/$(ConfigurationName)-$(Platform).